bin/delete_ecr_images

```diff
@@ -13,7 +13,7 @@ images = JSON.parse(json_output)['imageDetails']
 images_to_delete = []
 images.each do |i|
   date_pushed = DateTime.strptime(i['imagePushedAt'].to_s, '%s')
```

Comment:
should this be the following to ensure consistency with the new standard?
`DateTime.strptime(i['imagePushedAt'].to_s, '%s').in_time_zone`